.travis.yml

```diff
@@ -1,33 +1,22 @@
 language: node_js
 sudo: false
 node_js:
-  - 0.12
-  - 4
-  - 6
   - 8
 services:
   - redis-server
+  - mongodb  
 script:
   - npm run coverage
 after_success:
   - npm run publish-coverage
 env:
-  global:
-    - CC=gcc-4.8
-    - CXX=g++-4.8
-    - MONGODB_VERSION="3.4.9"
+  - CXX=g++-4.8
```

Comment:
please keep mongo here. Feel free to update the version number.
The mongodb service from Travis is very old.